### ISSUE TITLE
Extract out web3 typescript declarations info its own package?

### ISSUE THREAD

**Issue by rej156** [ConsenSys/ethql#84]
Mono repo with lerna and yarn workspaces?

**Comment by pcardune** [comment]
There already is a package: https://www.npmjs.com/package/@types/web3. it's just buggy 🙁

**Comment by raulk** [comment]
Yeah, rather than submitting a stream of PRs which will take time to be merged, we decided to fix the Web3 typings locally for now. Once we feel they are stable, we’ll submit a PR to DefinitelyTyped to get them merged upstream.
